tests/lib/thrift/ThriftTest/src/TestUtils.hpp

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2022 Young Mei
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include <cstdio>
+#include <iostream>
```

Comment:
Typically in C++ headers, `<iosfwd>` is used, because it requires less work by the compiler.